base/linalg/dense.jl

```diff
@@ -9,6 +9,12 @@ const DOT_CUTOFF = 128
 const ASUM_CUTOFF = 32
 const NRM2_CUTOFF = 32
 
+# Generic cross-over constant based on benchmarking on a single thread with an i7 CPU @ 2.5GHz
+# L1 cache: 32K, L2 cache: 256K, L3 cache: 6144K
+# The constant below is exactly 2^21 = 2M
+# This constant should ideally be determined by the actual CPU cache size
+const ISONE_CUTOFF = 2097152
```

Comment:
Instead of the comment 2 lines above, you could directly assign `2^21` to the constant here (with `# 2MB` as comment if you like).